packages/manager/src/features/Databases/DatabaseDetail/DatabaseBackups/RestoreFromBackupDialog.tsx

```diff
@@ -115,6 +115,6 @@ export const RestoreFromBackupDialog: React.FC<CombinedProps> = (props) => {
   );
 };
 
-const enhanced = compose<CombinedProps, Props>(withPreferences());
+const enhanced = compose<CombinedProps, Props>(withPreferences);
 
 export default enhanced(RestoreFromBackupDialog);
```

Comment:
Let's use the hooks in this component too.

```suggestion
export default RestoreFromBackupDialog;
```